ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -339,8 +341,14 @@ public void init() {
         (req, res) -> {
           String body = util.safeGetBody(req);
           List<MetacardChanges> metacardChanges = GSON.fromJson(body, METACARD_CHANGES_LIST_TYPE);
-
-          UpdateResponse updateResponse = patchMetacards(metacardChanges, getSubjectIdentifier());
+          Set<String> sources =
+              Optional.ofNullable(req.queryParams("source"))
```

Comment:
❗ Sources are read only.  But you could change this to support `CatalogStore` IDs instead which can be updated.